deploy/helm/openfaas/Chart.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+description: Enable Kubernetes as a backend for OpenFaaS (Functions as a Service)
```

Comment:
Why did we need to copy-paste this in? Probably would be good to put the reason in here, or maybe in a readme.md file in this directory?

Reply:
Mainly for:
- embedding it as part of Magda. e.g. control the namespace it's deployed into
- make it helm 3 compatible.
Will update the readme.md

Reply:
Updated readme.md